tests/pytests/conftest.py

```diff
@@ -31,6 +31,10 @@ def sct2e(self, *args):
         return 0.1
     def cidfrm(self, *args):
         return (2000, "Test_Body_Frame", True)
+    def furnsh(self, *args):
+        if not hasattr(self, '_loaded_kernels'):
+            self._loaded_kernels = []
+        self._loaded_kernels.append(args)
```

Comment:
A simple solution to see what order things are being furnished. Might cause stateful errors if used all over a test module but only used in one test for now. Maybe find a better solution?

Reply:
Can this be replaced by mocking the furnsh and unload functions, then checking what they were called with and in what order?

Reply:
I would argue that this solution is simpler, I imagine for the same reason that we haven't replaced the rest of SimpleSpice with mocks.